### PR TITLE
feat(foundation/colors): add foundation/colors

### PR DESCRIPTION
**Pull request by andresin87**
## Foundation/Colors
**TASK**:
https://github.com/SUI-Components/sui-components/issues/1438

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Only
- [ ] Refactor

### CODE REVIEW

**Comment by turolopezsanabria** [comment]
**1/4** This section must be the first item of the Sidebar
1st of menu


**2/4** The white decoration is not needed
decoration


**3/4** There's an extra space for the Pantone name that we don't need, and it is forcing an extra height. Please remove it
pantone name


**4/4** The font size is ilegible at current size. It has to be 14px
font-size

**Comment by jcger** [comment]
what about tokens instead of foundation?

File: components/foundation/colors/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# FoundationColors
+
+> Description
```
Comment (adriadoz) [inline components/foundation/colors/README.md:3]: a little description can help

**Review by adriadoz — APPROVED** [review]

**Comment by turolopezsanabria** [comment]
> tead of foundation?

Valid point, I sent a poll in Slack for all to comment on this topic

For me **Foundations** include more than just **Tokens**
- Principles
- Colours
- Typography
- Icons
- Illustration
- Motion
- Spacing
- Media Queries
- Grid (Rhythm)
- Accessibility
- etc

**Review by nucliweb — APPROVED** [review]
👏

**Pull request closed without merging**